packages/core/src/index.html

```diff
@@ -236,6 +236,7 @@
     </div>
 </div>
 <!-- End Temporary Mobile Landscape UI Blocker -->
+<div id="popup_root" class="popup-root"></div>
```

Comment:
This can be removed. It shouldn't overlap with modal, they should be mutually exclusive, and it is currently only implemented in Trader. Perhaps we can rework how this is implemented?